src/app/documentation/demos/code/code-highlight-snippet.ts

```diff
@@ -6,15 +6,16 @@
 import {Component} from "@angular/core";
 
 const HTML_EXAMPLE = `
+<!-- "&#123;" left curly brace / "&#125;" right curly brace -->
 <pre>
     <code clr-code-highlight="language-css">
-        .some-component {
+        .some-component &#123;
```

Comment:
how:
1. clone `clarity-seed`, npm install
2. edit `about.component.html`, insert the code highlight example html
3. run `ng serve`
4. open browser, visit http://localhost:4200/about; watch console
=> see the error.

I am a VMware employer and just read the documentation and try all of them to have a check. Just thinking about how to do G11N work on the site and to be more familiar with Clarity, I tried the documentation.